lib/dai-plugin-mcd/src/CdpTypeService.js

```diff
@@ -7,15 +7,13 @@ export default class CdpTypeService extends Maker.PublicService {
   }
 
   initialize(settings = {}) {
-    if (settings.cdpTypes) {
-      this._cdpTypes = settings.cdpTypes.map(ilk =>
-        this.getCdpTypeObject(ilk.currency)
-      );
-    }
+    this._cdpTypes = (settings.cdpTypes || []).map(ilk =>
```

Comment:
@tyler17 subtle but important tweak here: this ensures that even if `settings.cdpTypes` is not defined, `this._cdpTypes` will still be an empty array. that will avoid an error when calling `listCdpTypes`, because it will call `map` on the empty array instead of an undefined value